charts/incubator/amd-automatic-music-downloader/Chart.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: v2
+appVersion: "latest"
+dependencies:
+- name: common
+  repository: https://truecharts.org
+  version: 8.15.2
```

A bump here might be nice:
```suggestion
  version: 8.15.4
```